Backend/app/rider-platform/rider-app/Main/src/API/UI/HotSpot.hs

```diff
@@ -146,3 +166,18 @@ getPointAtDistance Maps.LatLong {..} distance bearing =
             (sin aRad * sin (distance / earthRadius) * cos lat1Rad)
             (cos (distance / earthRadius) - sin lat1Rad * sin lat2Rad)
    in Maps.LatLong (toDegrees lat2Rad) (toDegrees lon2Rad)
+
+getDistanceBetweenLatLong :: Maps.LatLong -> Maps.LatLong -> Double
+getDistanceBetweenLatLong latLong1 latLong2 =
```

Comment:
you can use this function `distanceBetweenInMeters` from `Kernel.Utils.CalculateDistance` for calculating distance between two points instead of defining a new one.

Reply:
done